app/profile/[[...username]]/page.tsx

```diff
@@ -1,58 +1,28 @@
-import { collection, query, where, getDocs } from "firebase/firestore";
-import { getServerSession } from "next-auth";
+import { getUsername, getUser } from "utils/getUser";
 
 import FeedPage from "../../../components/feed/FeedPage";
 import PageTitle from "../../../components/ui/PageTitle";
 import { formatFollowers } from "../../../lib/formatters";
-import { authOptions } from "../../../pages/api/auth/[...nextauth]";
-import db from "../../firebase";
+import { getFeedContent } from "../../../pages/api/feedContent/[id]";
 
 import type { FeedItemContent } from "../../../components/feed/FeedPage";
-
-// TODO SOPHIE FIX THIS (make it a utility function so we can use it in other places)
-const getUser = async () => {
-  const session = await getServerSession(authOptions);
-
-  if (!session) {
-    throw new Error("No session");
-  }
-
-  return session.user;
-};
-
-const getCurrentUsername = async () => {
-  const user = await getUser();
-  return user.id;
-};
-
-async function getData(username: string) {
-  // change to profile's username
-  const q = query(
-    collection(db, "feed_content"),
-    where("username", "==", username)
-  );
-  const qSnapshot = await getDocs(q);
-
-  return qSnapshot.docs.map(
-    (doc) => ({ id: doc.id, data: doc.data() } as FeedItemContent)
-  );
-}
+import type { User } from "next-auth";
```

Comment:
Delete this, they're not being used

Reply:
Run `yarn run next lint -d app -d components -d cypress -d lib -d pages -d types --report-unused-disable-directives error` to get a list of all the unused imports (ignore the `Error: Unused eslint-disable directive`)